examples/boids_spatial3D/src/main.cu

```diff
@@ -300,25 +328,25 @@ int main(int argc, const char ** argv) {
         env.newProperty("POPULATION_TO_GENERATE", 32768u);
 
         // Environment Bounds
-        env.newProperty("MIN_POSITION", -0.5f);
-        env.newProperty("MAX_POSITION", +0.5f);
+        env.newProperty("MIN_POSITION", -1.5f);
```

Comment:
This was previously -.5 to .5 to have a 1 unit cube of volume centred on zero. Might be worth checking with @mondus on his preference (via slack?)

Reply:
Yes this along with the other parameter changes need some thought. This volume provides a sensible density for the number of agents present. The 1 unit cube essentially creates one giant flock immediately as there isn't enough room. The other parameter changes have similar impact.

Reply:
Can't you instead change the interaction radius, rather than the env? Or would this mean other weights need changing?

Reply:
(My personal preference however is to make all units roughly equivalent to metres, normalised is hard to work with)

Reply:
Possibly, the other weights have been changed already anyway. Switching to metres would definitely necessitate increasing the env size though, would have to get very creative to fit 32k birds in a 1m box :D

Reply:
Maybe Reynolds was mistaken and it's a flock of flies.